CMakeLists.txt

```diff
@@ -114,21 +120,21 @@ endif ()
 
 # Configure the paths for output
 if (NOT DEFINED CMAKE_LIBRARY_OUTPUT_DIRECTORY)
-    set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+    set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib")
```

Comment:
Well, it can't affect other projects, because we placed it in a `if (NOT DEFINED CMAKE_LIBRARY_OUTPUT_DIRECTORY)`. Since `WolkConnect-Cpp` will set it's own `CMAKE_LIBRARY_OUTPUT_DIRECTORY`, `WolkSDK-Cpp` will just use that.